src/domain/flow-system/src/entities/shared/flow_key.rs

```diff
@@ -34,13 +35,23 @@ impl FlowKey {
 pub struct FlowKeyDataset {
     pub dataset_id: DatasetID,
     pub flow_type: DatasetFlowType,
+    pub options: CompactOptions,
```

Comment:
Logically this is unrelated to keys at all.
I need to think where to put this data, it's rather a part of flow configuration.
For now, pass the defaults to the service.

Reply:
Removed as well